rules/id/id-jawi.js

```diff
@@ -0,0 +1,50 @@
+﻿( function ( $ ) {
+	'use strict';
+	var jawiKeyboard = {
+		id: 'id-jawi',
+		name: 'Jawi keyboard',
+		description: 'Jawi keyboard for Indonesian Wikisource',
+		date: '2023-09-21',
+		URL: 'https://www.mediawiki.org/wiki/Help:Extension:UniversalLanguageSelector/Input_methods/jawi-keyboard',
+		author: 'Benny Lin, Pungkas, Michael',
+		license: 'GPLv3',
+		version: '1.0',
+		contextLength: 3,
```

Comment:
As far as I understand this input method is a 1:1 input method. The output of key strokes does not depend on previous input(context). hence concxtLength can be 0. Please validate

Reply:
it does depend on the previous input, as in 'gh', 'th', 'kh' characters